src/NovelRT/Audio/AudioService.cpp

```diff
@@ -319,7 +327,7 @@ std::string AudioService::getALError() {
       return std::string("The requested operation resulted in OpenAL running out of memory.");
     }
     default: {
-      return nullptr;
+      return std::string("");
```

Comment:
any particular reason you didn't just do `return "";` here?

Reply:
To match us doing `std::string("...")` in the other cases